render.yaml

```diff
@@ -18,24 +19,8 @@ services:
           name: squidex_mongo
           type: pserv
           property: hostport
-      - key: EVENTSTORE__TYPE
-        value: MongoDB
       - key: IDENTITY__ADMINEMAIL
```

Comment:
You can also remove the admin email because the initial admin can be configured with a setup screen.